web/ui/module/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+MODULE_LIST=(codemirror-promql)
+
+build-module() {
+  for module in "${MODULE_LIST[@]}"; do
+    cd "${module}"
+    echo "building ${module}"
+    npm run build
```

Comment:
We use yarn for the React app in Prometheus. We should standardize on either npm or yarn for the whole repo, IMO. I don't mind which one though. Maybe slight preference for npm these days.

It doesn't have to be in this PR, but it would be good to do soon.

Reply:
yup let's align everything. Makes totally sense. I feel the same than you, but no big opinion here. Probably using npm since it's used in lezer-promql as well.

Reply:
I will do it in another PR. It would be simpler

Reply:
PR available here: https://github.com/prometheus/prometheus/pull/9198